### ISSUE TITLE
Hide archived events/projects from Collective page

### ISSUE THREAD

**Issue by gustavlrsn** [opencollective/opencollective#6495]
## User story 
As a Collective admin, I would like to hide events/projects that I've archived from the main Collective page, since by archiving I am intending to put it away or delete the event/project (but deletion is not available if there are orders to the event/project).

## Best solution for this problem
1) Filter out archived projects and events from the Collective page.
2) Make archived events/projects available on the dedicated pages for projects and events, perhaps under a "Archived" tab or filter, so that they are still available for transparency/posterity

**Comment by jdaviescoates** [comment]
Yeah, this is really urgently needed imho. It seems at present the only way to hide and event is to delete it and loose all the associated data. By default past events should not be shown on the profile page, nor on the /events/ page.

I'm organising weekly events using Open Collective and this is going to quite quickly get out of hand - people will find it really hard and confusing to find the correct week to book with all the past events being constantly displayed.

**Comment by Betree** [comment]
> Oh, I don't know if it was just a caching or timezone thing or if you've already made improvements, but now I see that past events are listed at the end and are marked as Past Events

Yoy may have faced a cache issue, but this has been the default for quite a long time now: https://github.com/opencollective/opencollective-frontend/pull/4677.

> As a Collective admin, I would like to hide events/projects that I've archived from the main Collective page, since by archiving I am intending to put it away or delete the event/project (but deletion is not available if there are orders to the event/project).

It's important to note that different collectives have different expectations there. Some will argue that archiving is a way to officially close events (the end date [might not be enough](https://github.com/opencollective/opencollective/issues/6367)), but for transparency/history reasons they don't want to hide them.

One thing for sure: we'll need to keep easy access for admins in one way or another (e.g. for refunding a contribution). #6385 could provide an easy solution for this.

**Comment by derberg** [comment]
yeah, I see too many events 😅  and archived ones should not be shown by default.

best would be:
- main collective landing page - show only future
- then under https://opencollective.com/asyncapi/events show by default all except of archived
- there can be a button that says, show archived or whatever

no hiding, full transparency 🤷🏼 but archiving always means "disappear" from main list as it is the same as `close`. This was for example the long waited feature for Discussions in GitHub that for unknown reason was questioned by GH for months, but they finally did it, as this is the most natural way. If something becomes archived/closed -> obsolete, hide it by default.

thanks for reading 🙏🏼

**Comment by shannondwray** [comment]
https://github.com/opencollective/opencollective/issues/2199 - Duplicate issue
